_data/simple-icons.json

```diff
@@ -2757,6 +2757,11 @@
             "hex": "E7352C",
             "source": "https://www.espressif.com/"
         },
+        {
+            "title": "Etcd",
+            "hex": "419EDA",
```

Comment:
It seems you took this color from the SVG, is that correct? While the choice seems sensible I wondered if you looked around for any indication of the brand color of Etcd?

Reply:
I pick the brand color of etcd from `https://cncf-branding.netlify.app/projects/etcd/`, I think it's convinced.🧐 and i'll change the source address to this website as the same.

Reply:
That is perfect!

That wasn't the source when I reviewed this right :sweat_smile: If it was, I just glossed right over the color...